readme_renderer/rst.py

```diff
@@ -25,26 +24,9 @@
 
 class ReadMeHTMLTranslator(HTMLTranslator):
 
-    def depart_image(self, node):
-        uri = node["uri"]
-        ext = os.path.splitext(uri)[1].lower()
-        # we need to swap RST's use of `object` with `img` tags
-        # see http://git.io/5me3dA
-        if ext == ".svg":
-            # preserve essential attributes
-            atts = {}
-            for attribute, value in node.attributes.items():
-                # we have no time for empty values
-                if value:
-                    if attribute == "uri":
-                        atts["src"] = value
-                    else:
-                        atts[attribute] = value
-
-            # toss off `object` tag
-            self.body.pop()
-            # add on `img` with attributes
-            self.body.append(self.starttag(node, "img", **atts))
+    # We need to swap RST's use of `object` with `img` tags
```

Comment:
Can you update this comment to mention that we're override the base class to *not* output svg with `object`?

Reply:
Done.